.devcontainer/postCreateCommand.sh

```diff
@@ -16,6 +16,7 @@ python3 -m venv dev/venv
 pip install invoke
 invoke update
 invoke setup-dev
+invoke frontend-install
```

Comment:
Won't we add this task to the setup dev?

Reply:
You don't really need the frontend deps to do backend dev so my idea was to only install when it is needed.

Reply:
Yeah, maybe it's worth spearing them  like setup-dev-frontend and setup-dev-backend and then let the setup-dev do both. I guess there are a lot of beginners which are not sure what they need and later ask why they get errors changing something on the frontend.